tests/integration.rs

```diff
@@ -15,22 +15,6 @@
 use core::convert::TryFrom;
 extern crate webpki;
 
-static ALL_SIGALGS: &[&webpki::SignatureAlgorithm] = &[
```

Comment:
When you revert this change, please all add a comment about keeping it in sync with the list in the command line utility.